src/System.Management.Automation/project.json

```diff
@@ -156,6 +156,7 @@
                 "System.Diagnostics.StackTrace": "4.0.1",
                 "System.Diagnostics.TraceSource": "4.0.0",
                 "System.Dynamic.Runtime": "4.0.11",
+                "System.IO.FileSystem": "4.0.2-beta-24412-03",
```

Comment:
We currently don't take any dependencies on preview packages (that would be the first precedent).
We would like to avoid going this route.
@nanalakshmanan the PR is currently on hold, until `dotnet` team will release a final package.